pkg/network/cluster_config_test.go

```diff
@@ -58,10 +58,20 @@ func TestValidateClusterConfig(t *testing.T) {
 	cc.ClusterNetwork[0].CIDR = "192.168.2.0/23"
 	haveError(cc, "CIDRs 192.168.0.0/20 and 192.168.2.0/23 overlap")
 
+	cc = *ClusterConfig.DeepCopy()
+	cc.ClusterNetwork[1].HostPrefix = 0
+	res := ValidateClusterConfig(cc)
+	g.Expect(res).Should(BeNil())
```

Comment:
maybe add a comment here noting that `cc.NetworkType` is `None` so we don't validate `HostPrefix`